src/fileoverview_comment_transformer.ts

```diff
@@ -17,6 +17,17 @@ import * as ts from './typescript';
 const FILEOVERVIEW_COMMENT_MARKERS: ReadonlySet<string> =
     new Set(['fileoverview', 'externs', 'modName', 'mods', 'pintomodule']);
 
+/**
+ * Returns true if the given comment is a \@fileoverview style comment in the Closure sense, i.e. a
+ * comment that has JSDoc tags marking it as a fileoverview comment.
+ * Note that this is different from TypeScript's understanding of the concept, where a file comment
+ * is a comment separated from the rest of the file by a double newline.
+ */
+export function isFileoverviewComment(text: string) {
```

Comment:
maybe name it isClosureFileoverviewComment?

Reply:
Done.